Tools/clinic/clinic.py

```diff
@@ -1250,18 +1250,32 @@ def parser_body(
             parser_prototype = self.PARSER_PROTOTYPE_VARARGS
             parser_definition = parser_body(parser_prototype, '    {option_group_parsing}')
 
-        elif not requires_defining_class and pos_only == len(parameters) - pseudo_args and clinic.limited_capi:
-            # positional-only for the limited C API
-            flags = "METH_VARARGS"
+        elif clinic.limited_capi:
+            if not requires_defining_class and pos_only == len(parameters) - pseudo_args:
```

Comment:
Does it work if `pseudo_args` is not zero?

Reply:
I have no idea. I don't know what are pseudo args.